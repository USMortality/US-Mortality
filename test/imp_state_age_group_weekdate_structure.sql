SELECT
    DISTINCT jurisdiction
FROM
    deaths.imp_state_age_group_weekdate_structure;

SELECT
    DISTINCT age_group
FROM
    deaths.imp_state_age_group_weekdate_structure;

SELECT
    count(*)
FROM
    deaths.imp_state_age_group_weekdate_structure;