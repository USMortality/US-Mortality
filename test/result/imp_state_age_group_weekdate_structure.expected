SELECT
DISTINCT jurisdiction
FROM
deaths.imp_state_age_group_weekdate_structure;
jurisdiction
Alabama
Alaska
Arizona
Arkansas
California
Colorado
Connecticut
Delaware
District of Columbia
Florida
Georgia
Hawaii
Idaho
Illinois
Indiana
Iowa
Kansas
Kentucky
Louisiana
Maine
Maryland
Massachusetts
Michigan
Minnesota
Mississippi
Missouri
Montana
Nebraska
Nevada
New Hampshire
New Jersey
New Mexico
New York
New York City
North Carolina
North Dakota
Ohio
Oklahoma
Oregon
Pennsylvania
Puerto Rico
Rhode Island
South Carolina
South Dakota
Tennessee
Texas
United States
Utah
Vermont
Virginia
Washington
West Virginia
Wisconsin
Wyoming
SELECT
DISTINCT age_group
FROM
deaths.imp_state_age_group_weekdate_structure;
age_group
25-44 years
45-64 years
65-74 years
75-84 years
85 years and older
Under 25 years
SELECT
count(*)
FROM
deaths.imp_state_age_group_weekdate_structure;
count(*)
18468
